modules/programs/alacritty/default.nix

```diff
@@ -0,0 +1,234 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  defaults = import ./default-config.nix;
+  cfg = config.programs.alacritty;
+
+  mapAttrNamesRecursive' = f: set:
+    let
+      recurse = set:
+        let
+          g =
+            name: value:
+            if isAttrs value
+              then nameValuePair (f name) (recurse value)
+              else nameValuePair (f name) value;
+        in mapAttrs' g set;
+    in recurse set;
+
+  toSnakeCase = replaceChars upperChars (map (c: "_${c}") lowerChars);
+
+  animationList = [
+    "Ease"
+    "EaseOut"
+    "EaseOutSine"
+    "EaseOutQuad"
+    "EaseOutCubic"
+    "EaseOutQuart"
+    "EaseOutQuint"
+    "EaseOutExpo"
+    "EaseOutCirc"
+    "Linear"
+  ];
+  actionList = [
```

Comment:
These variables should be separated by an empty line. Same with all the options; see the other modules for example.